lib/chorde/clients/memcached.py

```diff
@@ -301,7 +414,7 @@ def check_key(self, key, key_extra_len=0,
             len(key) + key_extra_len > self.server_max_key_length:
             raise self.MemcachedKeyLengthError("Key length is > %s"
                      % self.server_max_key_length)
-        if any(map(ord, key.translate(tmap))):
+        if 1 in key.translate(tmap):
```

Comment:
Maybe I am forgetting the objective here. Were we planning to drop support for Python 2? Or make it compatible with both versions?

Reply:
We're dropping support for 2